R/ci_generic.R

```diff
@@ -86,14 +93,19 @@
   # if we have adjusted SE, e.g. from kenward-roger, don't recompute
   # standard errors to save time...
   if (is.null(se)) {
-    stderror <- if (isTRUE(robust)) {
-      standard_error_robust(model, component = component, ...)
+    if (!is.null(vcov) || isTRUE(list(...)[["robust"]])) {
+      stderror <- standard_error(model,
+                                 component = component,
+                                 vcov = vcov,
+                                 vcov_args = vcov_args,
+                                 verbose = verbose,
+                                 ...)
     } else {
-      switch(method,
-        "kenward" = ,
-        "kr" = se_kenward(model),
-        "satterthwaite" = se_satterthwaite(model),
-        standard_error(model, component = component)
+      stderror <- switch(method,
+            "kenward" = se_kenward(model),
```

Comment:
you could omit `se_kenward(model)` here, `switch()` allows catching multiple values here.